src/uct/ib/ud/base/ud_iface.h

```diff
@@ -108,6 +112,7 @@ struct uct_ud_iface {
     struct {
         ucs_mpool_t          mp;
         unsigned             available;
+        unsigned             reserved;
```

Comment:
usually 'reserved' means 'reserved for future use', like placeholder for internal variables (for example in ucp_stream_poll_ep_t)

Reply:
renaming it to "quota"